frontend/src/components/buttons/Button.tsx

```diff
@@ -5,13 +5,19 @@ import styled from "styled-components";
 import { Colors } from "../../styles";
 
 const Button = styled(AButton)`
-  /* TODO: Change colors */
   background-color: ${Colors.primary};
   color: ${Colors.onPrimary};
+  border-radius: 4px;
+  border: none;
+  transition: 0.12s ease-in-out;
 
-  &:hover {
+  &:focus, &:visited, &:hover{
+    background-color: ${Colors.primary};
     color: ${Colors.onPrimary};
-    background-color: ${darken(0.2, Colors.primary)};
+  }
+
+  &:hover {
+    background-color: ${darken(0.07, Colors.hoverPrimary)};
```

Comment:
Nie potrzebujemy dodatkowego koloru na hover, wystarczy, że skorzystasz z z tego `primary` i przy pomocy darken zmienisz lekko jego odcień.

1. W przypadku jakiejkolwiek zmiany, później ciężej będzie znaleźć wszystkie miejsca w którym została wprowadzona zmiana (a tak to zmieniasz tylko w jednym miejscu - `Single Source of Truth`)

2. Kolor `hoverPrimary` nie jest generyczny i też nie wiadomo o co z nim chodzi. Czy jest to najechanie na przycisk, a może na tabelę? Teraz jeszcze to wszystko jest świeże, ale za jakiś czas zapomniałoby się jaka jest ogólna idea tego koloru.

Reply:
![image](https://user-images.githubusercontent.com/95183973/162742922-df4cb2d1-e2e3-4c29-a1c9-c4e6b1cd1a76.png)


![image](https://user-images.githubusercontent.com/95183973/162743136-4c1cc250-53e9-4345-a2d0-18b2c820dd18.png)